src/scripts.js

```diff
@@ -13,12 +13,12 @@ import Hydration from './Hydration';
 import Sleep from './Sleep';
 
 
-let userRepository = new UserRepository();
-
-userData.forEach(user => {
-  user = new User(user);
-  userRepository.users.push(user)
-});
+let userRepository = new UserRepository(userData).users;
+let currentUser = new User(userRepository[0]); // should keep an eye out for when the variable `user` is being used when it should be currentUser
```

Comment:
Great work refactoring this so it'll now work with the new format for our UserReopository class